src/utils/pgwire/src/error.rs

```diff
@@ -23,33 +23,33 @@ pub type PsqlResult<T> = std::result::Result<T, PsqlError>;
 #[derive(Error, Debug)]
 pub enum PsqlError {
     #[error("Startup Error when connect to session: {0}")]
-    StartupError(BoxedError),
+    StartupError(#[source] BoxedError),
 
-    #[error("PasswordError: {0}")]
-    PasswordError(IoError),
+    #[error("Invalid password")]
+    PasswordError,
 
     #[error("QueryError: {0}")]
-    QueryError(BoxedError),
+    QueryError(#[source] BoxedError),
 
     #[error("ParseError: {0}")]
-    ParseError(BoxedError),
+    ParseError(#[source] BoxedError),
 
     #[error("ExecuteError: {0}")]
-    ExecuteError(BoxedError),
+    ExecuteError(#[source] BoxedError),
 
-    #[error("{0}")]
+    #[error(transparent)]
     IoError(#[from] IoError),
 
-    #[error("{0}")]
     /// Include error for describe, bind.
+    #[error(transparent)]
     Internal(BoxedError),
 
-    #[error("Panicked when processing: {0}.
-This is a bug. We would appreciate a bug report at https://github.com/risingwavelabs/risingwave/issues/new?labels=type%2Fbug&template=bug_report.yml")]
+    #[error("Panicked when processing: {0}.\n
+This is a bug. We would appreciate a bug report at https://github.com/risingwavelabs/risingwave/issues/new?labels=type%2Fbug&template=bug_report.yml.")]
```

Comment:
Prefer not to add trailing dot after url. 😟

Reply:
From my own perspective, it's not about whether there's a URL. Since there's multiple sentences (even paragraphs) in the error message, maybe period is necessary to become more consistent. 😟

Reply:
I know I know, but e.g., 😟
<img width="907" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/e588344f-de54-43af-bc99-9fff6b09a367">

Reply:
https://developers.google.com/style/periods

Reply:
maybe we can `:\n<url>`

Reply:
Cool, will piggy-back in my next PR.